src/web/trykkomania/trykkomania.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Trykkomania
+level: 2
+author: Arve Seljebu
+---
+
+# Introduksjon {.intro}
+Denne oppgaven viser deg hvordan du kan lage et spill med JavaScript og dele det med vennene dine. Spillet har blitt kallt *Trykkomania* fordi det handler om å trykke på en ball flest mulig ganger før tiden renner ut.
+
+![](trykkomania.gif)
+
+# Steg 1: Åpne JS Bin {.activity}
+Vi skal bruke JS Bin (http://jsbin.com) for å lage spillet. Hvis du aldri har brukt JS Bin før, ta en titt på oppgaven [Hei JavaScript](../hei_js/hei_js.html).
+
+## Sjekkliste {.check}
+- Åpne adressen [jsbin.com](http://jsbin.com) i ett eget vindu.
+- Velg å vise **JavaScript** og **Output**.
+
+  ![](faner.png)
+
+- I **JavaScript** skriver vi koden.
+- I **Output** vises websiden.
+- Når du starter er begge fanene tomme.
+
+# Steg 2: Lage en ball {.activity}
+Vi skal bruke JavaScript til å lage innholdet på siden, altså vi lager *HTML* fra JavaScript. Du trenger ikke kunne noe spesielt om HTML, men om du ønsker lære om HTML, se [oppgaven Internett](../internett/internett.html).
+
+## Sjekkliste {.check}
+- Vi begynner med å lage en funksjon som heter `Ball`:
+
+  ```js
+  function Ball() {
+
+  }
+  ```
+
+- Inni funksjonen lager vi ett nytt `div`-element:
+
+  ```js
+  function Ball() {
+    var el = document.createElement('div')
```

Comment:
Jeg får "missing semicolon" i Jsbin på denne og alle andre linjer som skulle hatt semikolon.
Dersom det er en innstilling for å "slå det av" i jsbin, må det nesten informeres om, selv om jeg personlig foretrekker at man alltid bruker semikolon.